operator/apis/machinelearning.seldon.io/v1/seldondeployment_webhook.go

```diff
@@ -353,6 +352,8 @@ func (r *SeldonDeploymentSpec) DefaultSeldonDeployment(mldepName string, namespa
 				}
 			}
 		}
+
+		r.Predictors[i] = p
```

Comment:
Do we need this line?

Reply:
seems we do to get the predictor updated .Good GoLang question as to why...